pkg/client/cache/reflector.go

```diff
@@ -78,7 +84,13 @@ func NewNamespaceKeyedIndexerAndReflector(lw ListerWatcher, expectedType interfa
 // so that you can use reflectors to periodically process everything as well as
 // incrementally processing the things that change.
 func NewReflector(lw ListerWatcher, expectedType interface{}, store Store, resyncPeriod time.Duration) *Reflector {
+	return NewNamedReflector(getDefaultReflectorName("kubernetes/pkg/client/cache/", "kubernetes/pkg/controller/framework/"), lw, expectedType, store, resyncPeriod)
```

Comment:
What do you think about extracting these out into a constant array, for clarity?